src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/JurisdictionUserCreationRequest.java

```diff
@@ -23,3 +24,4 @@ public JurisdictionUserCreationRequest(@JsonProperty(value = "id") String id,
     }
 
 }
+*/

```

Comment:
Please delete this file rather than entirely comment it out

Reply:
Done